subsys/bluetooth/host/gatt.c

```diff
@@ -657,6 +657,11 @@ ssize_t bt_gatt_attr_write_ccc(struct bt_conn *conn,
 	/* Update cfg if don't match */
 	if (ccc->cfg[i].value != ccc->value) {
 		gatt_ccc_changed(attr, ccc);
+
+		if (IS_ENABLED(CONFIG_BT_SETTINGS_STORE_CCC_IMMEDIATELY) &&
+		    bt_addr_le_is_bonded(conn->id, &conn->le.dst)) {
+			bt_gatt_store_ccc(conn->id, &conn->le.dst);
```

Comment:
Could we defer this through the system workqueue to avoid increasing stack space in the RX thread? @Vudentz?

Reply:
Tested with https://github.com/zephyrproject-rtos/zephyr/pull/10408 and I need to increase the system workqueue stack size instead of RX thread stack.

Reply:
Looks like we trading the RX to the system wq, perhaps it is worth it since the system wq can get reused by others.

Reply:
+1, system workqueue has more users so overall it's better to have more stack there